packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -64,21 +74,24 @@ class ValueKey<T> extends LocalKey {
   String toString() => '[\'$value\']';
 }
 
-/// A [Key] that is only equal to itself.
+/// A key that is only equal to itself.
 class UniqueKey extends LocalKey {
+  /// Creates a key that is equal only to itself.
   const UniqueKey();
```

Comment:
it strikes me that having a const constructor here is a landmine. We should remove the "const".